src/themes.rs

```diff
@@ -8,186 +8,223 @@ use crate::util;
 
 lazy_static! {
     pub static ref SLICK: Theme = Theme {
-        idle_bg: "#424242".to_owned(),
-        idle_fg: "#ffffff".to_owned(),
-        info_bg: "#2196f3".to_owned(),
-        info_fg: "#ffffff".to_owned(),
-        good_bg: "#8bc34a".to_owned(),
-        good_fg: "#000000".to_owned(),
-        warning_bg: "#ffc107".to_owned(),
-        warning_fg: "#000000".to_owned(),
-        critical_bg: "#f44336".to_owned(),
-        critical_fg: "#ffffff".to_owned(),
+        native_separators: false.to_owned(),
```

Comment:
Pretty sure you can get rid of all these `to_owned`s
Try running cargo clippy.

Reply:
This should do the trick for 95% of it:
```
rustup component add clippy --toolchain nightly-x86_64-unknown-linux-gnu
cargo +nightly clippy --fix -Z unstable-options
```

Reply:
Hm yeah i left them there because i didn't know if they might be important.

Running your `clippy` command changed a lot of things in a lot of files, that's maybe for another PR. For now i'll only apply it to `themes.rs` and `util.rs`

Reply:
The other files were all fixed in master an hour or so ago so don't worry about it